### PR TITLE
Refactor readme and variables

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+# slapd global configuration
+default_global_config:
```
Comment (SlothOfAnarchy) [inline vars/main.yml:3]: why not also prefix this?

File: defaults/main.yml [hunk 2]
```diff
@@ -1,98 +1,12 @@
...
-olc_config:
+slapd_olc_config:
   dummy: value
-monitor_config:
+slapd_monitor_config:
   dummy: value
-mdb_config:
+slapd_mdb_config:
   dummy: value
-# slapd global configuration
```
Comment (SlothOfAnarchy) [inline defaults/main.yml:13]: why do you move the defaults?

File: vars/main.yml (hunk 1 above)
Comment (dasJ) [reply vars/main.yml:3]: It's a var that cannot be overriden from outside the role

File: defaults/main.yml (hunk 2 above)
Comment (dasJ) [reply defaults/main.yml:13]: They should not be modified from outside the role

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**